tests/positive/partitions/complex-mb.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2018 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package partitions
+
+import (
+	"github.com/coreos/ignition/tests/register"
+	"github.com/coreos/ignition/tests/types"
+)
+
+func init() {
+	// Everything and the kitchen sink
+	register.Register(register.PositiveTest, KitchenSinkMB())
+}
+
+func KitchenSinkMB() types.Test {
```

Comment:
Does only having the one partition specifying `start-mb` cover enough use cases (everywherre else is not included or 0)?